sdv/sequential/par.py

```diff
@@ -88,6 +113,102 @@ def get_parameters(self):
 
         return instantiated_parameters
 
-    def get_metadata(self):
-        """Return the ``SingleTableMetadata`` for this synthesizer."""
-        return self.metadata
+    def preprocess(self, data):
+        """Transform the raw data to numerical space.
+
+        For PAR, none of the sequence keys are transformed.
+        """
+        sequence_key_transformers = {sequence_key: None for sequence_key in self._sequence_key}
+        if self._data_processor._hyper_transformer.field_transformers == {}:
+            self.auto_assign_transformers(data)
+
+        self.update_transformers(sequence_key_transformers)
+        return super().preprocess(data)
+
+    def _fit_context_model(self, transformed):
+        LOGGER.debug(f'Fitting context synthesizer {self._context_synthesizer.__class__.__name__}')
+        if self.context_columns:
+            context = transformed[self._sequence_key + self.context_columns]
+        else:
+            context = transformed[self._sequence_key].copy()
+            # Add constant column to allow modeling
+            context[str(uuid.uuid4())] = 0
+
+        context = context.groupby(self._sequence_key).first().reset_index()
+        self._context_synthesizer.fit(context)
+
+    def _build_model(self):
+        return PARModel(**self._model_kwargs)
```

Comment:
This could be removed, I don't think there is a need of a `_build_model` method unless you have a `self._model_class` which makes this abstraction worth it.